BasePolicies/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Node.js Community Contributing Guide 1.0
+
+This document describes a very simple process suitable for most projects
+in the Node.js ecosystem. Projects are encouraged to adopt this whether they
+are hosted in the Node.js Foundation or not.
+
+The goal of this document is to create a contribution process that:
+
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how
+contributors can be involved in decision making.
+
+This document is based on much prior art in the Node.js community, io.js,
+and the Node.js project.
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Committer** is a subset of contributors who have been given write access to the repository.
+* A **TC** is a group of committers representing the required technical expertise to 
+resolve rare disputes.
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in doubt, log an issue, 
+any additional policies about what to include will be provided in the responses.
+
+Committers may direct you to another repository, ask for additional clarifications, and
+add appropriate metadata before the issue is addressed.
+
+Please be courteous, respectful, and follow the project's Code of Conduct.
+
+# Contributions
+
+Any change to resources in this repository must be through pull requests. This applies to all changes
+to documentation, code, binary files, etc. Even long term committers and TC members must use
+pull requests.
+
+No pull request can be merged without being reviewed. Committers should try to avoid merging
+their own pull requests and allow other committers to accept and merge them.
+
+For non-trivial contributions pull requests should sit for at least 36 hours to ensure that
+contributors in other timezones have time to review.
+
+The default for each contribution is that it is accepted once no committer has an objection.
+There is no additional "sign off" process for contributions to land. Once all issues brought
+by committers are addressed it can be landed by any committer. The majority of contributions
+should land this way without escalation to the TC.
+
+If a contribution is controversial and committers cannot agree about how to get it to land
+or if it should land then it should be escalated to the TC. TC members should regularly
+discuss pending contributions in order to find a resolution.
+
+# Becoming a Committer
+
+All contributors who land a non-trivial contribution should be on-boarded in a timely manner,
```

Comment:
This idea seems to need a bigger introduction because it's likely going to be a surprising concept for a lot of people. perhaps a lead-in with some basic philosophy about being open and embracing of people willing to invest time in the project through code or other means. Language also needs to make it clear that this is only something that is _offered_, not automatically given. Being added to a repo just because you submitted a PR without being asked is not something we want to see going on.